sites/avivator/src/utils.js

```diff
@@ -405,7 +405,7 @@ export async function getSingleSelectionStats3D({ loader, selection }) {
   const lowResSource = loader[loader.length - 1];
   const { shape, labels } = lowResSource;
   // eslint-disable-next-line no-bitwise
-  const sizeZ = shape[labels.indexOf('z')] >> (loader.length - 1);
```

Comment:
I believe this is correct for the number of pixels we display because we downsample in the z direction by the resolution to account for the fact that image pyramids do not sample in the z direction.  I suppose the correct behavior would be to check whether or not the displayed image is downsampled.  Does this make sense or is it totally off base?

Reply:
Ah, no, we are just using this to fetch data.  I think it would be correct to remove this then.

Reply:
I'm not sure I understand but it seems we are in agreement that this is a bug and fixed with erics changes?

Reply:
Yes.  I just didn't read far enough.  I must have added this bitshift thinking about accounting for the plane-skipping we do when rendering.  But for stats, it shouldn't matter.